applications/prompt-proto-service-latiss/values-usdfprod-prompt-processing.yaml

```diff
@@ -8,7 +8,7 @@ prompt-proto-service:
     repository: ghcr.io/lsst-dm/prompt-proto-service
     pullPolicy: Always
```

Comment:
Pre-existing issue, but should this be `IfNotPresent` instead of `Always`? Possibly a question for @dspeck1.

Reply:
IfNotPresent is preferred for prod as long as new tags or revision annotation updated (if tag same) when a new docker image is built.

Reply:
Thank you! This is changed to `IfNotPresent`